scripts/helpers/normalize_helpers.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
```

Comment:
The main reason for putting these functions into a separate file is so that I can load them in the spec. I was having trouble getting `require` to recognize the path to the script & I'm not sure why -- best guess is the lack of `.rb` extension. Is this the best way to handle this issue?

Reply:
this works